tests/lib/cli-engine/cli-engine.js

```diff
@@ -3507,27 +3620,35 @@ describe("CLIEngine", () => {
         });
 
         describe("with '--rulesdir' option", () => {
-            it("should use the configured rules which are defined by '--rulesdir' option.", () => {
-                const rootPath = getFixturePath("cli-engine/with-rulesdir");
-                const StubbedCLIEngine = defineCLIEngineWithInMemoryFileSystem({
-                    cwd: () => rootPath,
-                    files: {
-                        "internal-rules/test.js": `
+
+            const rootPath = getFixturePath("cli-engine/with-rulesdir");
```

Comment:
Would it be good to avoid mixing these directories with the one that gets copied in the top [before](https://github.com/eslint/eslint/blob/1df7fa69967cc514a541c363eb52d845e1b1d8f0/tests/lib/cli-engine/cli-engine.js#L83) and has its own `.eslintrc` files and `node_modules`?

Reply:
I think this is okay for now. My main goal was to do a one-to-one translation of existing tests. I'd rather not get into changing how they work at this point.